en/airframes/adding_a_new_frame.md

```diff
@@ -1,10 +1,10 @@
-# Adding a new Airframe Configuration
+# Adding a New Airframe Configuration
 
 PX4 uses canned configurations as starting point for airframes. Adding a configuration is straightforward: Create a new file which is prepended with a free autostart ID in the [init.d folder](https://github.com/PX4/Firmware/tree/master/ROMFS/px4fmu_common/init.d) and [build and upload](../setup/building_px4.md) the software.
 
 Developers not wanting to create their own configuration can instead customize existing configurations using text files on the microSD card, as detailed on the [custom system startup](../advanced/system_startup.md) page.
 
-## Airframe configurations
+## Airframe Configurations
 
 An airframe configuration consists of three main blocks:
```

Comment:
@bkueng The third block is "tuning gains" (not shown here). We don't have any comment/section on that block (while we do on config file and mixer file blocks) - where is that set/specified. Are there any other documents we  should link to explaining how to set these gains?

Reply:
It's this block:
```
	param set BAT_N_CELLS 2
	param set FW_AIRSPD_MAX 15
	param set FW_AIRSPD_MIN 10
	param set FW_AIRSPD_TRIM 13
	param set FW_R_TC 0.3
	param set FW_P_TC 0.3
	param set FW_L1_DAMPING 0.74
	param set FW_L1_PERIOD 16
	param set FW_LND_ANG 15
	param set FW_LND_FLALT 5
	param set FW_LND_HHDIST 15
	param set FW_LND_HVIRT 13
	param set FW_LND_TLALT 5
	param set FW_THR_LND_MAX 0
	param set FW_PR_FF 0.35
	param set FW_RR_FF 0.6
	param set FW_RR_P 0.04
```
And it's more than just tuning gains. It includes vehicle-specific parameter settings, for example the number of battery cells, or things like the gps position offset relative to the board. I think it makes sense to add a short section about that here and include a link to the tuning gains page.

For the mixer we can link to https://dev.px4.io/en/concept/mixing.html.

Reply:
Absolutely.